test/partiql-randomized-tests/src/test/kotlin/org/partiql/lang/randomized/eval/builtins/TimestampTemporalAccessorRandomizedTests.kt

```diff
@@ -1,121 +0,0 @@
-// package org.partiql.lang.randomized.eval.builtins
```

Comment:
(self-review) `TimestampTemporalAccessor` class was deleted in v1 branch so tests no longer relevant